src/scribe.js

```diff
@@ -179,10 +181,19 @@ define([
     return this.el.textContent;
   };
 
+  /**
+   * Replace the current undo manager. Can be used by undo manager plugins or directly by the user.
+   * @param {Object} undoManager The undo manager instance to be used by the current editor.
+   */
+  Scribe.prototype.setUndoManager = function (undoManager) {
```

Comment:
I don't think it should be possible to change the UndoManager outside initialisation. Not a blocker to merging but not something we should support either.